plans/integration/main.fmf

```diff
@@ -10,6 +10,6 @@ prepare:
     how: install
     package: python3-pip
   - name: Install Requre
-    script: pip3 install typing-extensions>=3.7.4.3 requre
+    script: pip3 install "typing-extensions>=3.7.4.3" requre
```

Comment:
Wow, that's really it? Good find!

Reply:
it's definitely part of the problem, i.e. why `typing-extensions` are not updated:

```
16:23 | mprchlik > ouch
16:23 | mprchlik > $ pip3 install typing-extensions>=3.7.4.3 requre
16:23 | mprchlik > $ cat \=3.7.4.3
16:23 | mprchlik > Requirement already satisfied: typing-extensions in /usr/lib/python3.6/site-packages
16:23 | mprchlik > $
16:23 | mprchlik > :)
```

Reply:
Yep, fixed \o/

Now it's "just" a matter of Centos Stream 8 vs podman, that I can't fix.

Reply:
Nice, that's good! Hopefully that podman thing should be fixed soon too. One possible problem - even though we worked around the typing-extensions dependency in the tests by using pip, if a new version of typing-extensions is not built in the near future, tmt won't work on Centos Stream 8 once we merge this PR.

Reply:
True, I just believe this defect would be limited to places where nitrate is needed - export, import, convert? Core functionality should be left unaffected. Tests that don't depend on podman are green, which is promising.

I don't know whether it's an acceptable price for type annotations.

Reply:
> Hm, that does not sound well. It would mean that nitrate integration would not work for folks running on rhel-8 and centos-stream-8?

Correct. But I got an idea I'd like to try as a workaround, and I'd like to see results from Centos Stream 8 to see more affected pieces.

Reply:
Wait a minute: the issue appears in test, pytest fails to import `requre` because of a conflict between a library `requre` needs (some `git` package). Import of `nitrate` did not appear to cause any issues, it's the `git` Python package that's causing this mayhem - does `tmt` have anything to do with this package outside of tmt tests? Because if not, then the only issue are tests, which can be "patched" by calling `pip3` to install updated `typing-extensions`, and export/import/etc. should be completely unaffected.

Reply:
> Import of `nitrate` did not appear to cause any issues, it's the `git` Python package that's causing this mayhem - does `tmt` have anything to do with this package outside of tmt tests?

No, `tmt` is not using or depending on the `git` Python package. So it seems ok.